example/example-etcd-cluster-clusterwide.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: "etcd.database.coreos.com/v1beta2"
+kind: "EtcdCluster"
+metadata:
+  name: "example-etcd-cluster"
+  # Adding this annotation make this cluster managed by clusterwide operators
+  # namespaced operators ignore it
+  annotations:
```

Comment:
Can we combine this to `example-etcd-cluster.yaml` and add a comment for this?

Reply:
oops my bad, was what I wanted to do